ibis/expr/types.py

```diff
@@ -41,6 +41,9 @@ class Expr(object):
 
     """
 
+    def _type_display(self):
+        return type(self).__name__
```

Comment:
Any reason not to just make this the default implementation? Are there code paths the depend on it being `array` or `array-sort` etc?